### PR TITLE
Big fat, horrible, hacky implementation to allow free text responses.

### PR DESCRIPTION
**Pull request by rtibbles**

### CODE REVIEW

File: src/widgets/free-text-input.jsx [hunk 1]
```diff
@@ -0,0 +1,580 @@
...
+            ref="input"
+            value={this.props.currentValue}
+            onChange={this.handleChange}
+            className={classNames(classes)}
+            type={this._getInputType()}
+            examples={this.examples()}
+            shouldShowExamples={this.shouldShowExamples()}
+            onFocus={this._handleFocus}
+            onBlur={this._handleBlur}
+            {...this.props} />;
```
Comment (66eli77) [inline src/widgets/free-text-input.jsx]: "{...this.props}" this line throws the error message "Uncaught TypeError: Converting circular structure to JSON"

Comment (rtibbles) [inline src/widgets/free-text-input.jsx]: Damn, need to work out how to add in the object properties properly into TextArea, any ideas?

File: src/components/text-area.jsx [hunk 2]
```diff
@@ -0,0 +1,54 @@
...
+
+    getDefaultProps: function() {
+        return {
+            value: ""
+        };
+    },
+
+    render: function() {
+        return <textarea
+            {...this.props}
```
Comment (66eli77) [inline src/components/text-area.jsx:20]: I think this "{...this.props}" together with the "{...this.props}" inside free-text-input.jsx  creates a self reference and according to this http://stackoverflow.com/questions/21234436/uncaught-typeerror-converting-circular-structure-to-json
the self reference creates a infinite loop in json

Comment (rtibbles) [inline src/components/text-area.jsx:20]: Yeah, I assumed this is where the problem lies, I guess I just have to set the properties on it explicitly?

Comment (66eli77) [inline src/components/text-area.jsx:20]: yeah, maybe in  free-text-input.jsx, delete the "{...this.props}"  and explicitly set 'cols' and 'rows'

**Comment by 66eli77** [comment]
fixed! 
63q9t3

**Pull request merged**